OpenAPI/kiota/using.md

```diff
@@ -700,3 +706,13 @@ Available for commands: download, search.
 ```bash
 kiota <command name> --version beta
 ```
+
+### `--disable-ssl-validation (--dsv)`
+
+The disableSSLValidation is an option that allows users to disable SSL certificate validation. When this option is set to true, the SSL certificates presented by HTTPS servers of the openAPI Description File will not be validated. This can be useful in development or testing environments with self-signed certificates.
```

Comment:
```suggestion
The disable SSL validation is an option that allows users to disable SSL certificate validation. When this option is set to true, the SSL certificates presented by HTTPS servers of the OpenAPI Description File will not be validated. This can be useful in development or testing environments with self-signed certificates.
```

Reply:
Accepted